tests/integration.rs

```diff
@@ -4356,6 +4372,68 @@ mod tests {
             handle_child_output(r, &output);
         }
 
+        #[test]
+        #[cfg(target_arch = "x86_64")]
+        fn test_virtio_balloon() {
+            let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+            let guest = Guest::new(Box::new(focal));
+
+            let kernel_path = direct_kernel_boot_path();
+
+            let api_socket = temp_api_path(&guest.tmp_dir);
+
+            let guest_memory_size_gb = 4;
+
+            //Let's start a 4G guest with balloon occupied 2G memory
+            let mut child = GuestCommand::new(&guest)
+                .args(&["--api-socket", &api_socket])
+                .args(&["--cpus", "boot=4"])
```

Comment:
Do you need 4 vCPUs?

Reply:
> Do you need 4 vCPUs?

Emmm, 1 vCPU should be enough after changing to start a 4G guest with balloon occupied 2G memory and using `stress --vm 25 --vm-keep --vm-bytes 1G --timeout 20`.  And after changing to 1 vCPU, the test works, I will change to 1 cpu later. :)  PS: at the beginning, I used 1 vCPU + 10G/32G balloon_size/total_mem, but it took too long time to trigger an oom. That's why I increased to 4 vCPUs.

Reply:
Makes sense!